### ISSUE TITLE
[Request] htop-vim

### ISSUE THREAD

**Issue by guiltiest-gear** [chaotic-aur/packages#2610]
### Package:

[htop-vim](https://aur.archlinux.org/packages/htop-vim)

### Purpose:

Vanilla htop with a vim keybindings patch

### Benefits:

Although htim already exists in the chaotic AUR repos, htop-vim is more up to date with the current release of htop

### Copyright:

GPL2

### Expected Interest:

Some

### Already available?

No

### Unique request?

Yes

### Banned package?

No

**Comment by xiota** [comment]
Are `htop-vim` and `htim` functionally interchangeable?  Should both be included?  If so, can you describe the differences that make them both relevant?

**Comment by guiltiest-gear** [comment]
I personally think `htop-vim` should replace `htim` since the latter does throw the error of:
```
WARNING: /home/jade/.config/htop/htoprc specifies configuration format
         version v3, but this htop binary only supports up to version v2.
         The configuration file will be downgraded to v2 when htop exits.
```
It's pretty annoying having my pre-existing settings overwritten by `htim`, while `htop-vim` works as intended. I can however understand keeping `htim` as apparently it has a visual mode, but `htop-vim` does not. Some people may find use in the additional mode, but I can't necessarily speak for the users of the `htim` package, especially considering I'm not one of those users. I don't think the pro of `htim`'s visual mode outweighs the con of downgrading the htoprc file.

**Tl;Dr Replace `htim` with `htop-vim`.**

**Comment by xiota** [comment]
I just tested `htim`.  Saw the error.  Cannot get visual mode to work.  Will look into replacing it outright with `htop-vim`.

**Comment by REmerald** [comment]
Btw, since we have `htop-git`, maybe it would to a good idea to also add `htop-vim-git`?

**Comment by xiota** [comment]
What benefit does the `-git` version have?  I'd rather not add packages that no one actually needs because it's another package that could break and expend maintainer effort to fix.

**Comment by REmerald** [comment]
@xiota, idk, but the same reason ppl need `htop-vim` i guess. Personally, I think `htop-vim*` > `htop*`, so if someone needs `htop-git` they would probably benefit more from `htop-vim-git`. But that's just my thoughts, and personally I'm just gonna use `htop-vim` for now. So, it's up to you.

Btw, I just found, it was added in #278, but is absent now, for some reason.

**Comment by Technetium1** [comment]
If anyone feels like chasing a mystery, it seems `htop-vim-git` did exist at one point in `#278` as linked above but it did vanish without a trace…
`git log --all -p -G "htop-vim($|\s.*)"`
```diff
commit 3906152cbc374117636b58ae098e982b799ca81a (HEAD -> main, origin/main, origin/HEAD)
Author: xiota <xiota@users.noreply.github.com>
Date:   Thu Jul 13 02:03:18 2023 -0700

    Replace htim with htop-vim
    Closes #2610

diff --git a/ufscar-hpc/hourly.1.txt b/ufscar-hpc/hourly.1.txt
index 63bb3bb..7acdc21 100644
--- a/ufscar-hpc/hourly.1.txt
+++ b/ufscar-hpc/hourly.1.txt
@@ -1677,6 +1677,9 @@ jumpapp
 # Issue 2607
 zapret-git:https://aur.archlinux.org/zapret-git.git

+# Issue 2610
+htop-vim
+
 # Issue 2619
 ranger-git
```

**Comment by xiota** [comment]
#278 was closed by OP.  Package was probably never really added.  Could have been a test build, their own personal build, or some other mistake.

The request predates templates and doesn't include any info about why OP wanted it.  No one, including OP, requested a rebuild or otherwise noticed that the package was "missing" for years.

**Comment by Technetium1** [comment]
@xiota metrics get deleted after a while, so not seeing it there doesn't give much hindsight. On the other hand, there does seem to be issues with metrics in general now.

**Comment by xiota** [comment]
Mainly want to avoid adding packages that aren't really wanted or needed.  Thinking about if the package breaks later.  Someone will have to figure out why, how to fix it, convince the maintainer to update, add or update interferes, etc.

Anyone who does want `htop-vim-git`, please explain why at #278, including benefits and use cases that differentiate it from `htop-vim`.

**Comment by xiota** [comment]
Metrics for anyone who's interested.
```
htop-git # 4
htop-vim # 9
```
